stdlib/influxdata/influxdb/alerts/alerts.flux

```diff
@@ -0,0 +1,9 @@
+package alerts
+
+builtin check
+
+write = (tables=<-) => tables |> to(bucket: "system")
+
+from = () => from(bucket: "system")
```

Comment:
I think `alerts.from` will automatically pivot the data now if I understand things correctly.